.travis.yml

```diff
@@ -10,8 +10,14 @@ jobs:
     - &docker-generation
       stage: Docker generation
       language: generic
-      services: docker
       name: tue-env
+      before_install:
```

Comment:
Have you tried the updated version from APT?
```yaml
addons:
  apt:
    packages:
      - docker-ce
```

Reply:
Yeah. It doesn't install the latest version. That's why I did this.